snapd-glib/snapd-client.c

```diff
@@ -1337,6 +1338,54 @@ snapd_client_get_auth_data (SnapdClient *self)
     return priv->auth_data;
 }
 
+
+/**
+ * snapd_client_get_notices_async:
+ * @client: a #SnapdClient.
+ * @cancellable: (allow-none): a #GCancellable or %NULL.
+ * @callback: (scope async): a #GAsyncReadyCallback to call when the request is satisfied.
+ * @user_data: (closure): the data to pass to callback function.
+ *
+ * Asynchronously get notifications that have occurred / are occurring on the snap daemon.
+ *
+ * Since: 1.29
```

Comment:
1.65

Reply:
And other cases below.

Reply:
Yeah, wasn't sure which version should I put... Fixed.